docker/Dockerfile.base

```diff
@@ -10,26 +10,25 @@ ENV DEBIAN_FRONTEND=noninteractive
 
 # hadolint ignore=DL3005,DL3008
 RUN apt-get update && \
-    apt-get install --no-install-recommends -y \
-      ca-certificates \
-      curl \
-      iptables \
-      iproute2 \
-      iputils-ping \
-      knot-dnsutils \
-      netcat \
-      tcpdump \
-      conntrack \
-      bsdmainutils \
-      net-tools \
-      lsof \
-      linux-tools-generic \
-      sudo \
-   && update-ca-certificates \
-   && apt-get upgrade -y \
-   && apt-get clean \
-   && rm -rf  /var/log/*log /var/lib/apt/lists/* /var/log/apt/* /var/lib/dpkg/*-old /var/cache/debconf/*-old
+  apt-get install --no-install-recommends -y \
+  ca-certificates \
+  curl \
+  iptables \
+  iproute2 \
+  iputils-ping \
+  knot-dnsutils \
+  netcat \
+  tcpdump \
+  conntrack \
+  bsdmainutils \
+  net-tools \
+  lsof \
+  linux-tools-generic \
+  sudo \
+  && update-ca-certificates \
+  && apt-get upgrade -y \
+  && apt-get clean \
+  && rm -rf  /var/log/*log /var/lib/apt/lists/* /var/log/apt/* /var/lib/dpkg/*-old /var/cache/debconf/*-old
 
 # Sudoers used to allow tcpdump and other debug utilities.
-RUN useradd -m --uid 1337 istio-proxy && \
-    echo "istio-proxy ALL=NOPASSWD: ALL" >> /etc/sudoers
+RUN echo "istio-proxy ALL=NOPASSWD: ALL" >> /etc/sudoers
```

Comment:
This is the base used for pilot/proxy - real usage, we need the user

Reply:
Just reverted. Thanks!